script/shared/agent.ts

```diff
@@ -0,0 +1,147 @@
+import path from "path";
+import fs from "fs";
+
+import { Config } from "./config";
+import { defaultContextPath } from "./constants";
+import { Context, ContextHook } from "./context";
+import { extractByte32AddrFromBech32 } from "./utils";
+import { getContractInfo } from "./wasm";
+
+export type HplAgentConfig = {
```

Comment:
Can you import this from the SDK https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/nambrot/core-deployer-non-evm/node_modules/@hyperlane-xyz/sdk/src/metadata/agentConfig.ts#L390 ?